app/controllers/control.py

```diff
@@ -66,3 +70,24 @@ def mutate(cls, _, info, control_id, content):
         control.note = content
         db.session.commit()
         return control
+
+
+@app.route(
+    "/controllers/control-pictures-generate-presigned-urls", methods=["POST"]
+)
+@doc(
+    description="Génération d'url à durée limitée pour uploader les images d'un contrôle"
+)
+@with_authorization_policy(controller_only)
+@use_kwargs(
+    {
+        "control_id": fields.Int(required=True),
+        "nb_pictures": fields.Int(required=True),
+    },
+    apply=True,
+)
+def control_pictures_generate_presigned_urls(control_id, nb_pictures):
+    presigned_urls = S3Client.generated_presigned_urls_for_control(
+        control_id, nb_pictures
+    )
+    return jsonify({"presigned-urls": presigned_urls})
```

Comment:
thought(non-blocking): "presigned-urls" is neither snake case (for python) nor camel case (for js)